src/app/OperationalSessionSetup.cpp

```diff
@@ -411,7 +411,7 @@ void OperationalSessionSetup::PerformAddressUpdate()
     CHIP_ERROR err = LookupPeerAddress();
     if (err != CHIP_NO_ERROR)
     {
-        ChipLogError(Controller, "PerformAddressUpdate could not perform lookup");
+        ChipLogError(Controller, "PerformAddressUpdate could not perform lookup: %" CHIP_ERROR_FORMAT, err.Format());
```

Comment:
Collission with https://github.com/project-chip/connectedhomeip/pull/22335

Reply:
Aligned with the wording in that PR, so things will merge cleanly in either direction.